tests/src/tools/ReadyToRun.SuperIlc/Buckets.cs

```diff
@@ -153,14 +153,23 @@ private static string AnalyzeExecutionFailure(ProcessInfo process)
                         }
                         return line;
                     }
-                    else if (line.StartsWith("Unhandled Exception:"))
+                    else if (line.StartsWith("Unhandled exception", StringComparison.OrdinalIgnoreCase))
```

Comment:
Note: I think this used to work, but there was a relatively recent change in the unhandled exception formatting in console output, so it has probably broken it here.